streaming/base/shuffle/__init__.py

```diff
@@ -8,11 +8,13 @@
 
 from streaming.base.shuffle.naive import get_shuffle_naive
 from streaming.base.shuffle.py1b import get_shuffle_py1b
+from streaming.base.shuffle.py1br import get_shuffle_py1br
```

Comment:
Brainstorming: should we update the `shuffle_block_size (int)` datatype in [dataset.py](https://github.com/mosaicml/streaming/blob/v0.5.2/streaming/base/dataset.py#L283) to `shuffle_block_size (int | Tuple[int, int], optional) = None` to allow user to control the width of the range? For example,
- If user selects `py1b` as a shuffling algorithm, and does not provide `shuffle_block_size`, streaming would set `shuffle_block_size=1 << 18`.
- If user selects `py1b` as a shuffling algorithm, and does provide a `shuffle_block_size`, streaming would set `shuffle_block_size` to user specific value.
- If user selects `py1br` as a shuffling algorithm, and does not provide `shuffle_block_size`, streaming would set `shuffle_block_size` in range between [0.75 * 1 << 18, 1.25 * 1 << 18) .
- If user selects `py1br` as a shuffling algorithm, and does provide a `shuffle_block_size` as `int`, streaming would set `shuffle_block_size` in range between [0.75 * shuffle_block_size, 1.25 * shuffle_block_size).
-  If user selects `py1br` as a shuffling algorithm, and does provide `shuffle_block_size` as `Tuple[int, int]`, streaming would set `shuffle_block_size` in range between [value1, value2) .

Reply:
I think as per the short discussion on slack people really didn't want more knobs to turn in streaming. so it wasn't just about limiting the number of args, but also just not exposing functionality to users that nobody will need / will confuse people. in experiments the range of 0.75->1.25 seemed to work fine so imo we should just stick with that?

Reply:
Sure. Sounds good. We will re-iterate if there is a request from users for a knob